README.md

```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
+| <a href="https://drawkit.com/product/ecology-environment-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479444-40aa7a75-b8df-4736-a4f0-8001cd362f5b.png' alt="ecology_&_environment_free_illustrations" /></a> | [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit   |
+| <a href="https://drawkit.com/product/ecology-nature-free-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194713342-0a5264a2-1ebe-4497-9dee-f7136f29033e.png' alt="ecology_&_nature_free_illustrations" /></a> | [Ecology & Nature](https://drawkit.com/product/ecology-nature-free-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
+| <a href="https://www.drawkit.com/product/free-education-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964105-d0659d45-1c5f-4a31-8f29-2c82333232fe.png' alt="education_icons_free_illustrations" /></a> | [Education Icons](https://www.drawkit.com/product/free-education-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit   |
 | <a href="https://craftwork.design/downloads/error-404-illustrations/"><img width='700' src='https://user-images.githubusercontent.com/92253071/194913244-f990a85a-62ad-4ce7-9326-a2c0683f4809.png' alt="Error_404_free_illustrations" /></a> | [Error 404](https://craftwork.design/downloads/error-404-illustrations/)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • Sketch<br> • PNG<br> • SVG<br><br> ✌️Author <br> Craftwork |
 | <a href="https://drawkit.com/product/finn-character-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479227-c995cc31-2074-43e0-84f6-006ab06b54d2.png' alt="Finn_Character_free_illustrations" /></a> | [Finn Character](https://drawkit.com/product/finn-character-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit |
 | <a href="https://craftwork.design/downloads/flow-lava-constructor/"><img width='700' src='https://user-images.githubusercontent.com/74038190/195008950-4f5b442b-c22e-4ff0-9793-ad0edd7231c5.png' alt="Flow_Lava_free_illustrations" /></a> | [Flow Lava](https://craftwork.design/downloads/flow-lava-constructor/)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • Sketch<br> • PNG<br> • SVG<br><br> ✌️Author <br> Craftwork |
+| <a href="https://www.drawkit.com/product/food-drink"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964905-fdc1c359-3337-464f-8632-8bf4965af8e5.png' alt="food_&_drink_icons_free_illustrations" /></a> | [Food & Drink Icons](https://www.drawkit.com/product/food-drink)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit   |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195569620-ae39b498-c7f6-4b87-942b-7e6c7a343a50.png`